src/screens/RightDrawer/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { createDrawerNavigator } from '@react-navigation/drawer'
+import { createStackNavigator } from '@react-navigation/stack'
+
+import Screens, { AppStackParams } from '../../lib/screens'
+import { gurbaniScreen } from '../Gurbani'
+import { searchScreen } from '../Search'
+
+import DrawerContainer from './DrawerContainer'
+
+const { Screen, Navigator } = createDrawerNavigator()
+const { Screen: StackScreen, Navigator: StackNavigator } = createStackNavigator<AppStackParams>()
+
+const screens = [ gurbaniScreen, searchScreen ]
```

Comment:
These screens technically aren't in the right drawer,  they're the content in the middle. The left and right drawers should push/overlay the centre content.